data/homepage.yaml

```diff
@@ -7,10 +7,20 @@ banner:
   redhat_logo_alt: Red Hat logo
   welcome_alt: Welcome to Ansible
   platform_learn: Learn about Red Hat Ansible Automation Platform
+events:
+  section_icon: /assets/images/calendar-cyan.svg
+  icon_alt: Ansible community event icon
+  title: Events
+  intro: 
+  url: 
 ansible:
   section_icon: /assets/images/automation-cyan.svg
   icon_alt: Ansible automation icon
   image_alt: Ansible community collaboration
+  mission: Click here to learn more about our mission
+  mission_url: link://slug/mission-statement
+  join: Click here to join the community
+  join_url: https://docs.ansible.com/community.html
```

Comment:
These two are reported as suspicious links in violation of WCAG rules. Please see here for more info: https://webaim.org/standards/wcag/checklist#sc2.4.4

Reply:
I believe it's the "click here" wording that triggers the WCAG alert. It's vague about where users are going and makes it harder for users with assistive tech like screen readers. This resource seems to have a good explanation: https://www.linkedin.com/pulse/main-reasons-why-we-shouldnt-use-click-here-link-text-toby-osbourn